app/src/main/java/com/e/salesapp/LoginActivity.java

```diff
@@ -20,7 +20,7 @@ public void onClickSignUpBtn(View v){
     }
 
     public void onClickConfirmBtn(View v){
-        Intent mainIntent = new Intent(this, CategoriesActivity.class);
+        Intent mainIntent = new Intent(this, MainActivity.class);
```

Comment:
the name for this intent is no representative please change instead of some name more understandable